engine/runtime-with-instruments/src/test/java/org/enso/interpreter/test/instrument/AvoidIdInstrumentationTagTest.java

```diff
@@ -58,12 +58,12 @@ public void disposeContext() {
   }
 
   @Test
-  public void avoidIdInstrumentationInLambdaMapFunctionWithNoise() throws Exception {
+  public void avoidIdInstrumentationInLambdaMapFunctionWithFloor() throws Exception {
```

Comment:
@JaroslavTulach I hope that `floor` works equally well here in place of `noise`.

Reply:
I think so. Thanks for letting me know.